src/Hangman/HangmanGame.java

```diff
@@ -0,0 +1,208 @@
+package Hangman;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Scanner;
+
+public class HangmanGame {
+
+    private String[] board;
+
+    private static String word;
+
+    private static int lengthOfWord;
+
+    private static String[] lettersToGuess;
+
+    private static int incorrectGuesses = 0;
+
+    private static int correctGuesses = 0;
+    private static HashSet<String> lettersGuessed;
+
+
+
+    public HangmanGame(){
+        board = new String[]{"  ┌──┐",
```

Comment:
This string array in the constructor could be deleted, since it's never being used outside of this class itself. Any time we reference or manipulate the Hangman board; it's within the class itself. So the board in the constructor isn't needed. 

This hash set could also be moved from the constructor and be can be initialized where its already declared as a private member variable. Since this also isn't being used outside of this class itself.